packages/sampler/src/MoonstoneEnvironment/MoonstoneEnvironment.js

```diff
@@ -158,7 +158,7 @@ const StorybookDecorator = (story, config) => {
 			title={`${config.kind} ${config.story}`.trim()}
 			description={config.description}
 			locale={selectV2('locale', locales, 'en-US', globalGroup)}
-			textSize={boolean('large text', Config, (getPropFromURL('large text') === 'true')) ? 'large' : 'normal'}
+			textSize={boolean('large text', Config, getPropFromURL('large text') === 'true') ? 'large' : 'normal'}
```

Comment:
I'm not seeing how removing the set of parenthesis changes the result here. The ternary is still in the same place, functionally. Would you mind explaining in more detail?

Reply:
You're right ... I'm not sure why the result was different. I'll debug some more and either revise or explain.

Reply:
Not working now! Glad you noticed that. I wonder what other magic was in play.

Reply:
Back to my original analysis (which I didn't include on the ticket because I thought it was wrong) that `getPropFromUrl` is interfering. Storybook doesn't seem to reset the URL when reset is clicked but instead sometime after render (I didn't debug the specifics). Will close this PR and share my notes on in JIRA.